### PR TITLE
DROOLS-3224: [DMN Designer] Autolayout is not applied for 1.1 based DMN models

### PR DESCRIPTION
**Pull request by danielzhe**

### CODE REVIEW

**Comment by danielzhe** [comment]
jenkins execute full downstream build

**Comment by manstis** [comment]
I assume I fixed the issue?

Sadly a test would be nice.

**Comment by danielzhe** [comment]
@manstis Yes. Hum... ok, I'll make a test.

**Comment by danielzhe** [comment]
@manstis Test done.

File: kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/test/java/org/kie/workbench/common/stunner/core/graph/processing/layout/AbstractLayoutServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+public class AbstractLayoutServiceTest {
+
+    @Mock
+    private AbstractLayoutService layoutService;
+
+    @Mock
+    private Graph graph;
+
+    @Mock
+    Node n1;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/test/java/org/kie/workbench/common/stunner/core/graph/processing/layout/AbstractLayoutServiceTest.java]: Any reason you chose to make the fields above `private` but these not?

File: kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/test/java/org/kie/workbench/common/stunner/core/graph/processing/layout/AbstractLayoutServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    @Mock
+    Node n2;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/test/java/org/kie/workbench/common/stunner/core/graph/processing/layout/AbstractLayoutServiceTest.java]: Any reason you chose to make the fields above `private` but these not?

File: kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/test/java/org/kie/workbench/common/stunner/core/graph/processing/layout/AbstractLayoutServiceTest.java (hunk 1 above)
Comment (danielzhe) [reply kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/test/java/org/kie/workbench/common/stunner/core/graph/processing/layout/AbstractLayoutServiceTest.java]: No, I just forgot.

**Comment by danielzhe** [comment]
@manstis tests pushed.

**Comment by jomarko** [comment]
@danielzhe please link the PR with https://issues.jboss.org/browse/DROOLS-3219

**Comment by manstis** [comment]
@jomarko This PR covers a different issue to DROOLS-3219.

**Comment by jomarko** [comment]
@manstis understood, removed my wrong request for linking.

**Comment by danielzhe** [comment]
@manstis Just tested now too and is working. :)

**Pull request merged**